.buildkite/test-pipeline.yaml

```diff
@@ -126,7 +126,7 @@ steps:
   mirror_hardwares: [amd]
   commands:
     - bash ../.buildkite/download-images.sh
-    - pytest -v -s models -m vlm
+    - pytest -v -s -rx models -m vlm
```

Comment:
`-rx` in this context does not actually mean `-r` and `-x`, as described [here](https://docs.pytest.org/en/stable/how-to/skipping.html#how-to-use-skip-and-xfail-to-deal-with-tests-that-cannot-succeed) (yeah I know it is confusing).

Reply:
In any case, this will no longer be necessary as I'm working on using logprob-based tests instead of string-based ones. It's quite a hassle to convert the format of the text prompt from vLLM to HF in the test.